src/components/ScholarshipDetailCard.jsx

```diff
@@ -208,6 +209,10 @@ export default function ScholarshipDetailCard({ scholarship }) {
       </Box>
       <Chip
         component={Link}
+        href={genMailToLink({
+          subject: 'Report Scholarship Issue',
+          body: bugReport,
+        })}
```

Comment:
We need some way of identifying which scholarship the user is reporting an issue on. You might include the scholarship title in the subject and/or the scholarship link in the body.

Reply:
Also might be helpful to `bcc` the scholarship author's email.

Reply:
included all 3 suggestions